torchaudio/models/_wavernn.py

```diff
@@ -3,8 +3,15 @@
 import torch
 from torch import Tensor
 from torch import nn
+from .utils import load_state_dict_from_url
 
-__all__ = ["_ResBlock", "_MelResNet", "_Stretch2d", "_UpsampleNetwork", "_WaveRNN"]
+
+__all__ = ["_ResBlock", "_MelResNet", "_Stretch2d", "_UpsampleNetwork", "_WaveRNN", "_wavernn"]
+
+
+model_urls = {
+    '_wavernn': 'https://download.pytorch.org/models/_wavernn.pth',
```

Comment:
nit: It's conventional to add a FIXME comment here
```python
    # FIXME Replace URL by final one once determined
    '_wavernn': 'https://download.pytorch.org/models/_wavernn.pth',
```

Reply:
nit: once you know the actual URL, you can add it here without the underscore in the URL :)

Reply:
The actual URL has been updated.